### PR TITLE
Changes to add new language - Azerbaijani

### PR DESCRIPTION
**Pull request by woosal1337**
### Description

PR contains changes to add new language - Azerbaijani

### CODE REVIEW

**Review by Ferotiq — APPROVED** [review]

File: frontend/static/languages/_groups.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
       "english_commonly_misspelled"
     ]
   },
+  {
+    "name": "azerbaijani",
+    "languages": ["azerbaijani", "azerbaijani_1k"]
+  },
```
Comment (Miodec) [inline frontend/static/languages/_groups.json:17]: Please move the group lower. Groups are ordered roughly by the number of native speakers.

**Review by Miodec — CHANGES_REQUESTED** [review]

File: frontend/static/languages/_groups.json (hunk 1 above)
Comment (woosal1337) [reply frontend/static/languages/_groups.json:17]: I am sorry, @Miodec , but I doubt I have got you right. Could you elaborate on what you mean exactly?

Comment (Miodec) [reply frontend/static/languages/_groups.json:17]: Currently, this new group is right under english. Just move it lower in the file

Comment (woosal1337) [reply frontend/static/languages/_groups.json]: > Currently, this new group is right under english. Just move it lower in the file

Alright, @Miodec, I have moved it down under Belarusian. I hope everything should be fine now.

**Review by Miodec — APPROVED** [review]
Thanks

**Pull request merged**